lib/httplib/httpheaders.go

```diff
@@ -76,7 +88,11 @@ func getContentSecurityPolicyString(cspMaps ...map[string]string) string {
 
 	var cspStringBuilder strings.Builder
 	for _, k := range keys {
-		fmt.Fprintf(&cspStringBuilder, "%s %s; ", k, combined[k])
+		fmt.Fprintf(&cspStringBuilder, "%s", k)
```

Comment:
```suggestion
		cspStringBuilder.WriteString(k)
```

Reply:
Thanks, I'm not well acquainted with the `strings.Builder` api and somehow got in my head that this was the only way to write to it.